scripts/setup.py

```diff
@@ -92,7 +92,8 @@ def setup_state(args):
 
     s3_client = boto3.client("s3", region_name=REGION)
     s3_client.create_bucket(
-        ACL="private", Bucket=bucket_name,
+        ACL="private",
```

Comment:
No changes to this file, I ran `black` on the scripts directory and it also formatted this